src/index.ts

```diff
@@ -156,6 +156,15 @@ export interface SpannerOptions extends GrpcClientOptions {
   defaultTransactionOptions?: Pick<RunTransactionOptions, 'isolationLevel'>;
   observabilityOptions?: ObservabilityOptions;
   interceptors?: any[];
+  /**
+   * The Trusted Cloud Domain (TPC) DNS of the service used to make requests.
+   */
+  universe_domain?: string;
+  /**
+   * The Trusted Cloud Domain (TPC) DNS of the service used to make requests.
+   * Defaults to `googleapis.com`.
+   */
+  universeDomain?: string;
```

Comment:
We should add documentation mentioning that both are same , we support both cases, set either of these 2.